
  You can now view your Streamlit app in your browser.

  Local URL: http://localhost:8501
  Network URL: http://192.168.100.10:8501

  For better performance, install the Watchdog module:

  $ xcode-select --install
  $ pip install watchdog
            
2023-09-04 11:48:37.428 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:52.379 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:53.373 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:53.436 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:54.548 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:55.433 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:55.502 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:56.392 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:56.442 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:56.485 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:57.402 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:57.495 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:57.498 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:57.757 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:58.408 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:58.496 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:58.648 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:58.663 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:58.762 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:58.952 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:48:59.387 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:00.394 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:00.465 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:00.480 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:00.710 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:00.766 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:00.809 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:01.387 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:01.435 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:01.581 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:01.637 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:02.407 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:02.529 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:02.581 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:02.631 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:02.688 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:02.933 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:03.406 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:03.411 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:05.360 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:05.412 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
2023-09-04 11:49:08.371 Uncaught app exception
Traceback (most recent call last):
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
  File "/Users/galenmarchetti/hn-karma-hacking/app/streamlit_from_csv.py", line 85, in <module>
    story_df_with_comment_metrics['minutes_since_earliest_recent_comment']
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arraylike.py", line 116, in __truediv__
    return self._arith_method(other, operator.truediv)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/series.py", line 5526, in _arith_method
    result = ops.arithmetic_op(lvalues, rvalues, op)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/array_ops.py", line 218, in arithmetic_op
    res_values = op(left, right)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/ops/common.py", line 69, in new_method
    return method(self, other)
  File "/Users/galenmarchetti/.pyenv/versions/3.7.5/lib/python3.7/site-packages/pandas/core/arrays/timedeltas.py", line 627, in __rtruediv__
    f"Cannot divide {other.dtype} data by {type(self).__name__}"
TypeError: Cannot divide float64 data by TimedeltaArray
  Stopping...
